languages/go/exercises/concept/basics/.docs/after.md

```diff
@@ -0,0 +1,99 @@
+## Packages
+
+In Go an application is organized in packages. A package is a collection of source files located in the same folder. All source files in a folder must have the same package name at the top of the file. The package is preferred to be the same as the folder it is located in.
+
+```go
+package greeting
+```
+
+Standard library packages are imported using their name. Other packages are importet using the url of the package's location. A package can be given an alias when importing. This can be used to avoid package name conflicts:
```

Comment:
```suggestion
Standard library packages are imported using their name. Other packages are imported using the url of the package's location. A package can be given an alias when importing. This can be used to avoid package name conflicts:
```